src/index.html

```diff
@@ -6,9 +6,65 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+      >
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:wght@500&display=swap" 
+      rel="stylesheet"
+    >  
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
```

Comment:
remove this

Reply:
ok, thank you

Reply:
You haven't removed it :) Remove please )

Reply:
h1 was removed